applications/FluidDynamicsApplication/tests/CylinderTest/cylinder_test_probe1_ref.dat

```diff
@@ -1,13 +1,13 @@
 # Results for "element" with Id # 139 at position: x: 3.150768; y: 4.043068; z: 0
 # time VELOCITY_X VELOCITY_Y PRESSURE
-0.1 0.6594546997417607 0.03493609518313226 216.24431772568994
-0.2 0.5845361092516912 0.042820301585355845 -101.93346831850904
-0.3 0.6003540377856276 0.040676603139010954 25.351578684467615
-0.4 0.5982515934100076 0.04058831746014852 -5.6435709766301185
-0.5 0.595776533582532 0.040630643239784235 1.962412112334728
-0.6 0.5941394252911604 0.04060173260694352 0.07088092905660434
-0.7 0.5925809480842007 0.04058846805460195 0.5433236586945424
-0.8 0.5911340187348779 0.04058950607835889 0.4267377105323225
-0.9 0.5897872427616168 0.04059973181021452 0.45717767292052597
-1 0.588555135752233 0.040614339122423064 0.45119530599827923
-1.1 0.5874297726145277 0.04063544473195884 0.4542584323241518
+0.1 0.6594546997417409 0.03493609518311936 216.24431772568948
```

Comment:
can you explain why the results change?
From what I understood this PR is refactoring, which should not change the results?